resources/public/api/conf/1/application.raml

```diff
@@ -0,0 +1,35 @@
+#%RAML 1.0
+---
+
+title: Individuals
```

Comment:
This should probably be different here in income and over in employments. In the employments pull request we have it as `Individuals Employments` so maybe we should have `Individuals Income` here?

Reply:
makes sense